packages/text-bitmap/src/formats/TextFormat.ts

```diff
@@ -38,13 +38,13 @@ export class TextFormat
         for (const i in items)
         {
             // Extract item name
-            const name = items[i].match(/^[a-z]+/gm)[0];
+            const name = items[i].match(/^[a-z]+/gm)[0] as keyof BitmapFontData;
 
             // Extract item attribute list as string ex.: "width=10"
             const attributeList = items[i].match(/[a-zA-Z]+=([^\s"']+|"([^"]*)")/gm);
 
             // Convert attribute list into an object
-            const itemData = {};
+            const itemData: any = {};
```

Comment:
Maybe we do something stricter here?
```suggestion
            const itemData: {[name:keyof BitmapFontData]:string|number} = {};
```

Reply:
Had to make it this  😛 
`const itemData: Partial<{[value in keyof BitmapFontData]: string|number}> = {};`

Reply:
Typing this bit is kinda tricky. I guess it is parsing some unknown data.